tests/helpers/test_translation.py

```diff
@@ -712,3 +714,57 @@ async def test_get_translations_still_has_title_without_translations_files(
     assert translations == {
         "component.component1.title": "Component 1",
     }
+
+
+@pytest.fixture
+async def mock_integration(hass: HomeAssistant):
+    """Set up a mock notification service."""
+
+    async def test_service(call):
+        pass
+
+    hass.services.async_register(
+        "test_notify",
+        "send_message",
+        test_service,
+        schema=vol.Schema(
+            {
+                vol.Required("message"): cv.string,
+                vol.Optional("title"): cv.string,
+            }
+        ),
+    )
+
+
+async def test_translation_for_notify_service(
```

Comment:
Since we want to test the notify integration service descriptions I'd expect this test to be part of the notify integration (legacy) tests.